closed/OpenJ9.gmk

```diff
@@ -70,6 +70,15 @@ OPENJ9_VM_FILES := \
 	#
 
 ifeq (true,$(OPENJ9_ENABLE_DDR))
+  .PHONY : run-ddrgen
+  $(OUTPUT_ROOT)/vm/j9ddr.dat : run-ddrgen
+  run-ddrgen :
+	export $(EXPORT_MSVS_ENV_VARS) \
```

Comment:
This may not work well when `EXPORT_MSVS_ENV_VARS` is empty. Without arguments, `export` (on the versions of bash I tested) just dumps the existing environment variables. We could move `OPENJDK_VERSION_NUMBER_FOUR_POSITIONS` and `VERSION_MAJOR` into the export command to avoid that.